core/git.schema.go

```diff
@@ -0,0 +1,146 @@
+package core
```

Comment:
We should add a quick test for the new APIs [here](https://github.com/dagger/dagger/blob/b7f3f4fece8cf49da229f395e5914a1c4d01512b/core/integration/core.schema_test.go#L43-L64)

Reply:
OK, on it.

Reply:
Done.